users/spidey3/layer_rgb.c

```diff
@@ -474,9 +476,13 @@ void post_process_record_user_rgb(uint16_t keycode, keyrecord_t *record) {
 #endif
 
 #if defined(UNICODE_ENABLE) || defined(UNICODEMAP_ENABLE) || defined(UCIS_ENABLE)
```

Comment:
Just a heads up, `UNICODE_COMMON_ENABLE` could be checked here instead of all of these.